test/util/client.go

```diff
@@ -20,10 +23,10 @@ import (
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
-	"github.com/openshift/origin/pkg/cmd/util/tokencmd"
 	oauthapi "github.com/openshift/origin/pkg/oauth/apis/oauth"
 	oauthclient "github.com/openshift/origin/pkg/oauth/generated/internalclientset"
 	"github.com/openshift/origin/pkg/serviceaccounts"
+	userapi "github.com/openshift/origin/pkg/user/apis/user"
 	userclient "github.com/openshift/origin/pkg/user/generated/internalclientset"
```

Comment:
still dependent on internal apis and auth, i guess i'm not seeing how i copy this pattern.

Reply:
> still dependent on internal apis and auth, i guess i'm not seeing how i copy this pattern.

Hrm. All our integration tests are still on internal.  I don't think I've done anything special that's internal only.  Can you copy it and change the package imports to the externals?

Reply:
maybe.  it's bitten me in other places but i can try it.